frontend/src/components/workplan/TestSelectForm.js

```diff
@@ -7,33 +7,30 @@ import { getFromOpenElisServer } from "../utils/Utils";
 function TestSelectForm(props) {
   const mounted = useRef(false);
   const [tests, setTests] = useState([]);
-  const [defaultTestId, setDefaultTestId] = useState("");
-  const [defaultTestLabel, setDefaultTestLabel] = useState("");
+  const [selectedTestId, setSelectedTestId] = useState("");
 
   const handleChange = (e) => {
-    props.value(e.target.value, e.target.selectedOptions[0].text);
-  };
-
-  const getTests = (res) => {
-    if (mounted.current) {
-      setTests(res);
+    setSelectedTestId(e.target.value); // Update the selected test ID state
+    if (e.target.value !== "") {
+      const selectedOptionText = e.target.selectedOptions[0].text;
+      props.value(e.target.value, selectedOptionText);
     }
   };
 
   useEffect(() => {
     mounted.current = true;
-    let testId = new URLSearchParams(window.location.search).get(
-      "testId"
-    );
-    testId = testId ? testId : ""; 
     getFromOpenElisServer("/rest/tests", (fetchedTests) => {
-      let test = fetchedTests.find(test => test.id === testId);
-      let testLabel = test ? test.value : "";
```

Comment:
Can we just change the testLabel here to `testLabel = test ? test.value : "Select Test Type"`